.golangci.yml

```diff
@@ -3,19 +3,17 @@ linters-settings:
     check-shadowing: true
 linters:
   enable:
-    - deadcode
     - gofmt
     - goimports
     - revive
     - gosec
     - govet
     - ineffassign
-    - structcheck
     - typecheck
-    - varcheck
```

Comment:
There were three linters that the latest version of golangci-lint noted as deprecated and replaced with `unused`.  So I've done that here.